apps/meteor/tests/data/moderation.helper.ts

```diff
@@ -0,0 +1,16 @@
+import { api, credentials, request } from './api-data';
+
+export const reportUser = async (userId: string, reason: string) => {
+	const res = await request.post(api('moderation.reportUser')).set(credentials).send({
+		userId,
+		reason,
+	});
+	return res.body;
+};
+
+export const getUsersReports = async (userId: string) => {
+	const res = await request.get(api('moderation.user.reportsByUserId')).set(credentials).query({
+		userId,
+	});
+	return res.body;
+};
```

Comment:
try to avoid such functions it doest really help a lot something like that could be a bit. more usefull
```suggestion

const whatever = (url: string) => async (userId: string, reason: string) => {
	const res = await request.post(api(url)).set(credentials).send({
		userId,
		reason,
	});
	return res.body;
};

export const reportUser =whatever('moderation.reportUser')

export const getUsersReports = whatever('moderation.user.reportsByUserId'))
```


but still I would not omit the url from the tests

Reply:
Thank you, I will definitely keep a note of it.